internal/docs/docs-components/navigation.js

```diff
@@ -47,27 +85,35 @@ export default () => (
         </a>
       </li>
       <li>
-        <a href="/?url=manage/#/clients">
-          <Icon name="clients" color="grayLightest" size={20} />
-          <span>PoC Manage</span>
+        <a href="/#/playground">
```

Comment:
@landitus This links should be `/docs/#playground`

Reply:
Fixing it now! Thanks